### PR TITLE
Prepare for erc1559 2

### PR DESCRIPTION
**Pull request by oa-s**
pr does:
- removes transaction params, nonce, data, gaslimit from TransactionConfiguration, for simplifying updates;
- updates Configure transaction screen to handle timed updates of GasSpeed estimates;
- adds countdown times for refresh estimates data;
- remove recreating subviews when configure transaction screen get updated

### CODE REVIEW

**Review by hboon — APPROVED** [review]
Didn't check. Presuming it's a pure refactoring :P

**Comment by oa-s** [comment]
> Presuming it's a pure refactoring :P

nope, pr updates configure transaction flow to add support of timed updates of gas price estimates, updates configure transaction screen with removing recreating views, adds countdown timer to reload estimates

**Comment by oa-s** [comment]
pr is still wip, but good to check if it works right, will clean it before merge

**Comment by hboon** [comment]
> > Presuming it's a pure refactoring :P
> 
> nope, pr updates configure transaction flow to add support of timed updates of gas price estimates, updates configure transaction screen with removing recreating views, adds countdown timer to reload estimates

Will check. Can you then update the PR title and commit message then? Should remove "[Refactoring]" and describe the change with a 1-liner

File: AlphaWallet/Transfer/ViewModels/UpdateInViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        let isLoadingHidden = gasPriceEstimator.state
+            .map { state -> Bool in
+                switch state {
+                case .tick, .idle, .done: return true
+                case .loading: return false
+                }
+            }.removeDuplicates()
+
+        return .init(
+            text: Just(buildTitleString("Next update in:")).eraseToAnyPublisher(),
```
Comment (hboon) [inline AlphaWallet/Transfer/ViewModels/UpdateInViewModel.swift:48]: Remember to localise

**Review by hboon — COMMENTED** [review]
So it fetches the gas price every 15 secs, right? Looks good to me. Count down timer display is great.

- [ ] There are a couple of prints that are not "xxx" (look for `print("[LegacyEstimator]`). Probably should switch to `infoLog()` and friends

- [ ] The gas price estimator seems to continue running after the transaction confirmation actionsheet is closed by cancelling. This line runs: https://github.com/AlphaWallet/alpha-wallet-ios/blob/4ac2a7cfee74dc25afa829b6a32112437c8db78c/modules/AlphaWalletFoundation/AlphaWalletFoundation/GasPriceEstimator/Models/EtherscanGasPriceEstimator.swift#L27

**Comment by hboon** [comment]
Also, be good to mention this in commit 1-liner:

> adds countdown timer for refreshing gas price estimate

**Comment by hboon** [comment]
Looks ok previously. Need me to check anything, otherwise all good

**Comment by oa-s** [comment]
could be good to check flow one more time, to make sure all is good before merge

**Comment by oa-s** [comment]
@hboon added support to enter small values when editing gasPrice, gasLimit. seems good

**Pull request merged**